frontend/templates/troubleshooting/index.tsx

```diff
@@ -79,142 +82,174 @@ const Wiki: NextPageWithLayout<{
-            <HStack spacing={0}>
-               <Image
-                  src={mainImageUrl}
-                  onClick={onOpen}
-                  cursor="pointer"
-                  alt={title}
-                  htmlWidth={120}
-                  htmlHeight={90}
-                  objectFit="contain"
-                  borderRadius="4px"
-                  outline="1px solid"
-                  outlineColor="gray.300"
-                  marginRight="12px"
-                  display={{ base: 'none', sm: 'block' }}
-               />
-               <Modal isOpen={isOpen} onClose={onClose}>
-                  <ModalOverlay />
-                  <ModalContent
-                     width="auto"
-                     maxWidth="calc(100% - 64px)"
-                     background="none"
-                  >
-                     <VisuallyHidden>
-                        <ModalHeader>{title}</ModalHeader>
-                     </VisuallyHidden>
-                     <ModalCloseButton />
-                     <VisuallyHidden></VisuallyHidden>
-                     <ModalBody padding={0}>
-                        <Image
-                           src={mainImageUrlLarge}
-                           width="100%"
-                           height="auto"
-                           alt={title}
-                        />
-                     </ModalBody>
-                  </ModalContent>
-               </Modal>
-               <VStack alignItems="flex-start" spacing="0px">
-                  <HeadingSelfLink
-                     as="h1"
-                     fontSize="3xl"
-                     fontWeight="500"
-                     selfLinked
-                     id="top"
-                  >
-                     {wikiData.title}
-                  </HeadingSelfLink>
-                  <AuthorInformation
-                     lastUpdatedDate={lastUpdatedDate}
-                     authors={wikiData.authors}
-                     historyUrl={wikiData.historyUrl}
+         <Container fontSize="md" maxW="1280px">
+            <Flex
+               direction="column"
+               paddingInline={{ base: 0, sm: 4 }}
+               paddingBottom={8}
+               flexShrink="1"
+               id="main"
+            >
+               <Head>
+                  {metadata}
+                  <HreflangUrls urls={wikiData.hreflangUrls} />
+               </Head>
+               <HStack spacing={0} mt={{ base: 3, sm: 8 }} align="start">
+                  <Image
+                     src={mainImageUrl}
+                     onClick={onOpen}
+                     cursor="pointer"
+                     alt={title}
+                     htmlWidth={120}
+                     htmlHeight={90}
+                     objectFit="contain"
+                     borderRadius="md"
+                     outline="1px solid"
+                     outlineColor="gray.300"
+                     marginRight={3}
+                     display={{ base: 'none', sm: 'block' }}
```

Comment:
Not your fault, but I'd opt for a Show/Hide component instead of display none.

Mainly the dom node will not render thus no image will be loaded if it is not shown

Reply:
https://github.com/iFixit/react-commerce/pull/1816/commits/170549ba71521b796eb197cc8f47e5b41e52e0b8